pallets/runtime/testnet/src/runtime.rs

```diff
@@ -561,7 +561,8 @@ impl treasury::Trait for Runtime {
 }
 
 parameter_types! {
-    pub const MaxLegsInAInstruction: u32 = 20;
+    pub const MaxScheduledInstructionLegsPerBlock: u32 = 500;
+    pub const MaxLegsInAInstruction: u32 = 10;
```

Comment:
The article is "an". Maybe it's better omitted altogether.
```suggestion
    pub const MaxLegsInInstruction: u32 = 10;
```